packages/cdk-routing-split/test/cases/integration/delete.test.ts

```diff
@@ -0,0 +1,29 @@
+import * as when from '../../steps/when';
+import * as given from '../../steps/given';
+import * as then from '../../steps/then';
+import { createPK } from '../../../src/shared/make-keys';
+
+describe('When an api user', () => {
+    let addRequestBody: any;
+
+    beforeEach(() => {
+        addRequestBody = given.get_add_request_body();
+    });
+
+    it('calls delete with valid fields', async () => {
+        await when.we_invoke_add(addRequestBody);
```

Comment:
This test should call only delete end point, because that is under test. You should not call add end point just to add record to dynamo. This should be precondition and you do it by adding record directly over dynamo client.